doc/source/whatsnew/v1.2.0.rst

```diff
@@ -507,6 +507,7 @@ I/O
 - Bug in :class:`HDFStore` was dropping timezone information when exporting :class:`Series` with ``datetime64[ns, tz]`` dtypes with a fixed HDF5 store (:issue:`20594`)
 - :func:`read_csv` was closing user-provided binary file handles when ``engine="c"`` and an ``encoding`` was requested (:issue:`36980`)
 - Bug in :meth:`DataFrame.to_hdf` was not dropping missing rows with ``dropna=True`` (:issue:`35719`)
+- Bug in :func:`read_html` when parsing ``pathlib.Path`` object as html path (:issue:`37705`)
```

Comment:
Maybe a little bit more specific:
 :func:`read_html` was raising a ``TypeError`` when given a ``pathlib.Path`` object (:issue:`37705`)

Reply:
Edited accordingly! Thanks for the comments! @twoertwein